pkg/asset/manifests/vsphere/infrastructure.go

```diff
@@ -98,3 +101,16 @@ func GetInfraPlatformSpec(ic *installconfig.InstallConfig, clusterID string) *co
 	}
 	return &platformSpec
 }
+
+// vipsToCIDRs takes a single ip address and converts it to CIDR notation.
+func vipsToCIDRs(vips []string) []configv1.CIDR {
+	var cidrs []configv1.CIDR
+	for _, vip := range vips {
+		mask := "/32"
+		if strings.Contains(vip, ":") {
+			mask = "/64"
+		}
+		cidrs = append(cidrs, configv1.CIDR(vip+mask))
+	}
+	return cidrs
+}
```

Comment:
How bad is this? I have little idea what I'm doing, so please help!

Reply:
For IPv6 let's use `:128` if the desire is to add a single IP address

Reply:
Also, the logic of checking if IP address is IPv6 by checking if it contains `:` is quite error-prone, but for what we are doing it it should not matter (i.e. nothing is going to break if you add /32 when you wanted to add /128)

Reply:
Thanks! Updated.